website/docs/secrets/manage-secrets-ui.mdx

```diff
@@ -31,13 +31,49 @@ Here, you will be prompted to enter the `External Secret Name` and the `Target K
 
 It's important to note that the chosen `SecretStore` may be a cluster-scoped `SecretStore` ie: `ClusterSecretStore` or a namespace-scoped `SecretStore`.
 
+If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+
 ![Create new Secret](./imgs/create-secret-2.png)
 
-If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
 
 ![Create new Secret](./imgs/create-secret-3.png)
 
-If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
+#### Adding Secret Properties
+
+After you have choose your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:
```

Comment:
```suggestion
After you have chosen your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:
```

Reply:
Re `SecretPath` What is the information that the user needs before populating this field? I was looking at whether that info was in the UI or in the docs. I found it a bit later in the doc https://docs.gitops.weave.works/docs/secrets/manage-secrets-ui/#external-secret-details 

A user first facing the doc might require them to understand either from us or from other links.  It would be cool if they could be referenced as they first time appear or similar.